### PR TITLE
Enable Common Runtime Tests for Mono on Linux-Arm64 in CI

### PR DESCRIPTION
**Pull request by naricc**
Enable runtime tests on Linux Arm-64 with Mono.

### CODE REVIEW

File: eng/pipelines/runtime.yml [hunk 1]
```diff
@@ -410,6 +410,7 @@ jobs:
     platforms:
     - Android_x86
     - Android_arm64
+    - Linux_arm
```
Comment (steveisok) [inline eng/pipelines/runtime.yml:413]: @akoeplinger do you think we need Linux_arm debug on the libraries build?

Comment (akoeplinger) [reply eng/pipelines/runtime.yml:413]: No, the existing build should be enough just like we do for Linux_arm64 today: https://github.com/dotnet/runtime/blob/f007634b02c4484f60ead75a893eb7f89f615ccf/eng/pipelines/runtime.yml#L390

Comment (naricc) [reply eng/pipelines/runtime.yml:413]: Ok, switching it to use the release build of libraries.

File: eng/pipelines/common/templates/runtimes/run-test-job.yml [hunk 2]
```diff
@@ -274,7 +274,7 @@ jobs:
       - script: $(_msbuildCommand)
                 $(Build.SourcesDirectory)/src/mono/mono.proj
                 /t:PatchCoreClrCoreRoot
-                /p:Configuration=$(buildConfig)
```
Comment (safern) [inline eng/pipelines/common/templates/runtimes/run-test-job.yml:277]: I'm surprised this is needed, why are coreclr run jobs working currently?

Comment (safern) [reply eng/pipelines/common/templates/runtimes/run-test-job.yml:277]: Oh, this is Mono to patch the runtime, disregard.

File: eng/pipelines/runtime.yml [hunk 3]
```diff
@@ -691,6 +691,7 @@ jobs:
     runtimeFlavor: mono
     platforms:
     - OSX_x64
+    - Linux_arm64
```
Comment (safern) [inline eng/pipelines/runtime.yml:694]: Can we open an issue to track removing these steps and using the coreclr build-test steps instead? Of course the best will be to wait for: https://github.com/dotnet/runtime/pull/35783 to land.

**Comment by naricc** [comment]
I still need to add exclusions to issues.targets for linux arm64, but the logic is otherwise right and I thought I should start reviews.

File: eng/pipelines/runtime.yml (hunk 3 above)
Comment (naricc) [reply eng/pipelines/runtime.yml:694]: Ok, here it is: https://github.com/dotnet/runtime/issues/35979

Comment (safern) [reply eng/pipelines/runtime.yml:694]: Thanks.

**Review by safern — APPROVED** [review]

File: src/coreclr/tests/issues.targets [hunk 4]
```diff
@@ -1922,184 +1922,181 @@
...
-        <ExcludeList="$(XunitTestBinBase)/Directed/StructABI/StructABI/**">
+        <ExcludeList Include = "$(XunitTestBinBase)/Directed/StructABI/StructABI/**">
 	    <Issue>needs triage</Issue>
         </ExcludeList>
-        <ExcludeList="$(XunitTestBinBase)/Regression/VS-ia64-JIT/M00/b108366/b108366/**">
-	    <Issue>needs triage</Issue>
-        </ExcludeList>
-        <ExcludeList="**">
+        <ExcludeList Include = "$(XunitTestBinBase)/Regression/VS-ia64-JIT/M00/b108366/b108366/**">
 	    <Issue>needs triage</Issue>
```
Comment (safern) [inline src/coreclr/tests/issues.targets:2100]: `<Issue>` tags are not correctly indented.

**Comment by safern** [comment]
@jashook this will add another test run in Linux_ARM64, so just to be aware and on the lookout for any ARM64 slowdown on our test runs.

**Pull request merged**